addons/website/controllers/main.py

```diff
@@ -797,6 +799,83 @@ def seo_suggest(self, keywords=None, lang=None):
         xmlroot = ET.fromstring(response)
         return json.dumps([sugg[0].attrib['data'] for sugg in xmlroot if len(sugg) and sugg[0].attrib['data']])
 
+    @http.route(['/website/get_alt_images'], type='jsonrpc', auth="user", website=True)
+    def get_alt_images(self, res_id, res_model):
+        ret = []
+        record = request.env[res_model].browse(res_id)
+        editable_fields = record._editable_fields()
+        if not editable_fields:
+            return ret
+        for editable_field in editable_fields:
+            tree = html.fromstring(str(record[editable_field]))
+            for index, element in enumerate(tree.xpath('//img')):
+                decorative = element.attrib.get('role') == 'presentation'
```

Comment:
Is there a specific reason to move from `aria-hidden` to `role` ? I think you need to check if `role="presentation"`  OR `role="none"` here ?

Reply:
> Is there a specific reason to move from aria-hidden to role ?

I leave a message about it in the task if you want to have a look :)